homeassistant/components/glances/sensor.py

```diff
@@ -35,7 +35,7 @@
     'swap_use_percent': ['Swap used', '%', 'mdi:memory'],
     'swap_use': ['Swap used', 'GiB', 'mdi:memory'],
     'swap_free': ['Swap free', 'GiB', 'mdi:memory'],
-    'processor_load': ['CPU load', '15 min', 'mdi:memory'],
+    'processor_load': ['CPU load', '', 'mdi:memory'],
```

Comment:
This is a breaking change.

Reply:
Sorry if I haven't understood correctly the issue (not a software developer and this was my first PR) but I don't think it is a breaking change as it doesn't require any change in the frontend or in configuration.yaml and there aren't other components that depend on it. 

I used the modified version as a custom component for some time without problems.

"15 min" is not a meaningful measurement unit, anyway.

Reply:
There is no real measurement unit for CPU load anyway, but IMO the load over 15 minutes does have a meaning. So I wouldn't say it's meaningless or useless. It has its place. Maybe it should be part of the name if it really needs to be moved.

Reply:
> ...but I don't think it is a breaking change ...

It will break the automations and the longtime data recording of the users.

> "15 min" is not a meaningful measurement unit, anyway.

Well, there may be other options but in the current way most people know what the sensor is displaying. If people dislike the default settings of a sensor then those settings can be overwritten by customization directly in the frontend.

Reply:
> It will break the automations and the longtime data recording of the users.

These below are not my posts but I don't think my PR has something to break as the current values of the sensors are not usable in any automation now (unsure about longtime recording but doesn't seem wise to keep more than several days track of a sensor that updates 2 times per minute, anyway):
https://community.home-assistant.io/t/glances-and-formatting-processor-load/95661

How about the other frequently used load values such as 1 min load and 5 min load? I don't think that conceptually there is even the intent to define 2 more different measurement units in HA alongside "15 min" (I know that, currently, the sensor only uses 15 min load but it is very easy to define also the other two as they're exposed by Glances). 

I think 15 min should be included in the name of the sensor (like "CPU 15m load") and keep the sensor value as an integer as otherwise HA won't graph the values (now it doesn't). Usual values for load are something like "load 15m of 0.2" or "load 1m of 0.5" (without any unit).

I can rewrite the PR to (leave room also for 1m/5m load case someone wants it):

`'processor_load_15': ['CPU 15m load', '', 'mdi:memory'],`

and then:

            elif self.type == 'processor_load_15':
                # Windows systems don't provide load details
                try:
                    self._state = value['load']['min15']
```